tasks/price-consumer/index.js

```diff
@@ -1 +1,2 @@
 exports.readPriceFeedEns = require("./read-price-feed-ens.js")
+exports.readPriceFeed = require("./read-price-feed.js")
```

Comment:
Shouldn't the `read-price-feed` be kept in the plugin?

>  The only tasks that should be tasks are reading price feeds and funding LINK .

https://github.com/smartcontractkit/hardhat-starter-kit/issues/35#issue-984227226

Reply:
yes it could be a (separate) plugin as its a common task (assuming the consumer contract has a getLatestPrice() function, but for the purposes of this boilerplate project, we want to have all code contained in the project for new developers to see how it works, and to have a source of reference to copy and use for their own projects etc